### DISCUSSION TITLE
Recommended LHR OC Settings

### DISCUSSION THREAD

**Discussion by alt-savage** [NebuTech/NBMiner#549] [General]
Can the developers provide base recommended OC setting for the LHR cards they have tested ? Any insight into the working of Nvidia's LHR algorithm appreciated. Please share any LHR tuning strategies.

**Comment by ezMaverick** [comment]
Stable 85-86 for 3080 Ti Micron on HiveOS.
F: 80, C: 0, M: 2000, P: 280
